src/hub.html

```diff
@@ -759,8 +759,8 @@
                 cursorSuperHand: #cursor;"
         ></a-entity>
 
-        <a-entity matrix-auto-update class="vr-notice" follow-in-lower-fov="target: #player-camera; offset: 0 0 -0.8; angle: 39;" visible="false">
-            <a-entity hoverable slice9="width: 0.9; height: 0.2; left: 64; top: 64; right: 66; bottom: 66; opacity: 1.0; src: #action-button" 
```

Comment:
Did `#action-button` change to `#button` on purpose?

Reply:
yep. #button is now the grey, tintable buton. #action-button is the pink one.